packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -275,9 +284,16 @@ export const TranslationSidebar = forwardRef<
                 <div className="flex flex-col gap-6 pb-2">
                   {hasLanguageReadPermissions && (
                     <div className="flex flex-col gap-2">
-                      <h2 className="font-bold">
-                        {t('translate:translator_notes')}
-                      </h2>
+                      <div className="flex flex-row gap-2.5">
+                        <h2 className="font-bold">
+                          {t('translate:translator_notes')}
+                        </h2>
+                        {saveTranslatorNoteMutation.isLoading && (
+                          <em>
```

Comment:
It's better to use `<span class="italic">` than an `em` because `em` has semantic meaning that doesn't apply here

Reply:
Done.